docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml

```diff
@@ -29,16 +31,19 @@
   <emphasis>allowoverwrite</emphasis> <emphasis role="bold">] , [</emphasis>
   <emphasis>replicate</emphasis> <emphasis role="bold">] , [</emphasis>
   <emphasis> compress </emphasis> <emphasis role="bold">] , [</emphasis>
-  <emphasis> sourceCsvEscape </emphasis> <emphasis role="bold">])</emphasis>
-  <emphasis></emphasis></para>
+  <emphasis> sourceCsvEscape </emphasis> <emphasis role="bold">]</emphasis>,
```

Comment:
Independently my comment above there are some other parameters missing from the STD.File.SprayDelimited() and STD.File.fSprayDelimited() functions argument list: failIfNoSourceFile, recordStructurePresent, quotedTerminator.

They are exists in the table later.